src/elements/Label/Label.js

```diff
@@ -114,12 +131,6 @@ Label.propTypes = {
   /** Additional text with less emphasis. */
   detail: PropTypes.string,
 
-  /** An element type to render the 'detail' as (string or function). */
-  detailAs: PropTypes.oneOfType([
```

Comment:
Given we're removing the `detailAs`, should `detail` be a shorthand prop? So you can pass a string would be treated as `content` or you can pass an object as props or the element itself.